app/models/work.rb

```diff
@@ -0,0 +1,37 @@
+class Work < ApplicationRecord
+  validates :category, presence: true
+  validates :title, presence: true, uniqueness: true
+
+  has_many :votes, dependent: :destroy
+
+  def self.sort_by_category(category)
+    works_sorted_by_category = []
+    
+    Work.all.each do |work|
+      if work.category == category
+       works_sorted_by_category << work
+      end
+    end
+    return works_sorted_by_category
+  end
+
+  def self.popular(category: nil)
+    if category
+      works = Work.sort_by_category(category)
+    else
+      works = Work.all
+    end
+
+    return works.sort_by{ |work| -work.votes.count}
+  end
+
+  def self.media_spotlight
+    return popular.first
```

Comment:
Just a note for consistency (not a big deal): Above in `self.popular`, you invoke the other class methods with `Work.sort_by_category`, but here you invoke the class method with just `popular`. It may make more sense to be consistent: `Work.sort_by_category` and `Work.popular`, or `sort_by_category` and `popular` (PS: In your tests you consistently do `Work.popular` etc)